.github/workflows/manual.yml

```diff
@@ -0,0 +1,33 @@
+# SPDX-FileCopyrightText: 2024 The TetriQ authors
+#
+# SPDX-License-Identifier: CC0-1.0
+
+name: Manual deployment
+
+on:
+  push:
+    paths:
+      - "doc/**"
+
+jobs:
+  deploy:
```

Comment:
Deploy isn't relevant enough as a job name, it should be called "deploy_manual" or something else